signing/sawtooth_signing/secp256k1_signer.py

```diff
@@ -184,7 +111,7 @@ def _decode_pubkey(serialized_pubkey, encoding_format='hex'):
 
 
 def generate_identifier(pubkey):
```

Comment:
There's a call in core/sawtooth/client.py
https://github.com/hyperledger/sawtooth-core/blob/master/core/sawtooth/client.py#L405

I can't see where the result is ever used though.  Whether or not that client code should be pulled I think this PR has to not break that client.